terminus-ui/selection-list/src/selection-list-panel/selection-list-trigger.directive.ts

```diff
@@ -211,6 +213,18 @@ export class TsSelectionListTriggerDirective<ValueType = string> implements Cont
     return null;
   }
 
+  /**
+   * Calculates the height of the options
+   *
+   * Only called if at least one option exists
+   */
+  private get itemHeight(): number {
+    // Try to use the 2nd option in case the first option is blank or a filter etc. Fall back to the first item if needed.
+    const options = this.selectionListPanel.options.toArray();
+    const option = options[1] || options[0];
+    return option.elementRef.nativeElement.offsetHeight;
```

Comment:
Can the option height be defined by consumers?

Reply:
It pretty much already is. We are checking the rendered height, so if the consumer puts in `200px` tall content, that's what it will be. Our default height is just created by our CSS. So the template drives the height.